alacritty/windows/wix/alacritty.wxs

```diff
@@ -41,7 +41,7 @@
       <!-- Add context menu -->
       <Component Id="ContextMenu" Guid="449f9121-f7b9-41fe-82da-52349ea8ff91" Directory="TARGETDIR">
          <RegistryKey Root="HKCU" Key="Software\Classes\Directory\Background\shell\Open Alacritty here\command">
-            <RegistryValue Type="string" Value="[AlacrittyProgramFiles]alacritty.exe" KeyPath="yes" />
+            <RegistryValue Type="string" Value="[AlacrittyProgramFiles]alacritty.exe &quot;--working-directory&quot; &quot;%v.&quot;" KeyPath="yes" />
```

Comment:
Is it necessary to quote thhe `--working-directory` flag here? And does `\"` work or does it need to be `&quot;`?

Reply:
If omit `--working-directory`, for example:

```shell
alacritty C:/User/
```

will lead:

```
error: unrecognized subcommand 'C:/User/'

Usage: alacritty.exe [OPTIONS] [COMMAND]
```

And `\"` isn't standard escape character in XML, use `\"` will terminating string after `\`. That caused a error when wix processing in my test.

Reply:
Quoting  `--working-directory` and `%v`, follow the writing method of git shell and cmd. Without quotation, perhaps also work. But add quotes to prevent some unexpected problems

Reply:
As we all know, Windows handles command line parameters differently from Unix. So put it in a string to prevent parsing problems. VS Code even quotes the executable path itself.

Reply:
So now there are so many styles of writing:
1.
```
"[AlacrittyProgramFiles]alacritty.exe --working-directory %v"
```
2.
```
"[AlacrittyProgramFiles]alacritty.exe --working-directory &quot;%v&quot;"
```
3.
```
"[AlacrittyProgramFiles]alacritty.exe &quot;--working-directory&quot; &quot;%v&quot;"
```
4.
```
&quot;"[AlacrittyProgramFiles]alacritty.exe&quot; &quot;--working-directory&quot; &quot;%v&quot;"
```

which one do you choose?

I still recommend using quotation marks, especially 4.

This can avoid many problems caused by encoding or path spaces.

Reply:
Quoting the path is fine, as long as it is required and it doesn't work with a space in the path otherwise (that should be simple to test).

So if quotes around the path is required, then 2 would be the best option. There's no reason to quote something if it works without it. That just makes things unreadable.

Reply:
In my test, using single quotes not work:
1.
```
<RegistryValue Type="string" Value="'[AlacrittyProgramFiles]alacritty.exe' '--working-directory' '%v'" KeyPath="yes" />
```
But this work:
2.
```
<RegistryValue Type="string" Value='"[AlacrittyProgramFiles]alacritty.exe" "--working-directory" "%v"' KeyPath="yes" />
```
Even if the installation path set by msi has spaces (and working directory also has spaces), the following writing works:
3.
```
<RegistryValue Type="string" Value='[AlacrittyProgramFiles]alacritty.exe --working-directory %v' KeyPath="yes" />
```
This should be due to some patches in new versions of Windows. This is not guaranteed to work on older versions of Windows. 

Therefore, software such as VS Code will use quotation marks. This can be seen as a defensive programming technique.

In order to be consistent with the overall style of the project, I would like to ask you to choose a version between 2 and 3.